src/page/IFrame.page.ts

```diff
@@ -0,0 +1,48 @@
+import { $, browser, promise, ElementFinder, ExpectedConditions } from 'protractor';
+
+export class IFramePage {
+
+  private get frame1(): ElementFinder {
+    return $('#IF1');
+  }
+
+  private get frame1Title(): ElementFinder {
+    return $('.wpb_wrapper > h1');
+  }
+
+  private get mainPageTitle(): ElementFinder {
+    return $('#content > h1');
+  }
+
+  public get iFrame1Context(): promise.Promise<void> {
```

Comment:
this should not be a `get` function as you are doing an action instead of actually `getting` something the name should denote an action, something like `swithToFrame1`